packages/graph/hash_graph/lib/graph/src/shared/identifier/time/projection.rs

```diff
@@ -253,6 +256,9 @@ impl ToSchema for TransactionTimeKernel {
     }
 }
 
+/// The variable time of a [`TimeProjection`].
```

Comment:
Same as above but for `VariableTime`. These terms seem to be a lot more natural within our context